infra/aws/terraform/prow-build-cluster/terraform.canary.tfvars

```diff
@@ -40,22 +40,24 @@ node_min_size_blue     = 1
 node_max_size_blue     = 3
 node_desired_size_blue = 1
 
-node_taints_blue = [
-  {
-    key    = "dedicated"
-    value  = "kind-tests"
-    effect = "NO_SCHEDULE"
-  }
-]
-
-node_labels_blue = {
-  kind-exclusive = "true"
-}
-
-additional_node_group_tags_blue = {
-  "k8s.io/cluster-autoscaler/node-template/label/kind-exclusive" = "true"
-  "k8s.io/cluster-autoscaler/node-template/taint/dedicated"      = "kind-tests:NoSchedule"
-}
+# TODO(pkprzekwas): uncomment after migration to public subnet
+
+# node_taints_blue = [
+#   {
+#     key    = "dedicated"
+#     value  = "kind-tests"
+#     effect = "NO_SCHEDULE"
+#   }
+# ]
```

Comment:
I don't think we need this any longer by the way, we'll take a different approach for debugging this issue for now.